### PR TITLE
chore(drawer): upgrade to mdcw 2.x

### PR DESCRIPTION
**Pull request by lucasecdb**
related to #885

### CODE REVIEW

File: packages/drawer/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "url": "https://github.com/material-components/material-components-web-react.git"
   },
   "dependencies": {
-    "@material/drawer": "^1.0.1",
+    "@material/drawer": "^2.1.1",
     "@material/list": "^1.0.0",
```
Comment (moog16) [inline packages/drawer/package.json]: I think we should also update list here.

**Comment by moog16** [comment]
@lucasecdb I just checked https://github.com/material-components/material-components-web-react/issues/885...it looks like list has a breaking change. What is it that we need to fix?

**Comment by lucasecdb** [comment]
I think it's more like a ["usability" breaking change](https://github.com/material-components/material-components-web/blob/master/CHANGELOG.md#breaking-changes), the API is still the same, so I think we are good

**Comment by moog16** [comment]
Yes it looks like this was the PR:
https://github.com/material-components/material-components-web/pull/4568/files.
We are good!

**Comment by moog16** [comment]
i just rebased with master, and it messed this PR up :(

**Comment by lucasecdb** [comment]
oh let me try to fix it

**Comment by moog16** [comment]
can you update once more? I had to rebase again.

**Comment by moog16** [comment]
@lucasecdb pinging the PR :)

**Comment by lucasecdb** [comment]
I think I lost some commits in between those force pushes :-(

**Comment by moog16** [comment]
Ah yeah :( Sorry I made this so complicated.

**Comment by lucasecdb** [comment]
its ok, git is kinda hard when you are trying to rebase into a branch that has been updated

**Pull request merged**